<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserClass</class>
 <widget class="QMainWindow" name="UserClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>165</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="HelloLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ark Pixel 12px monospaced zh_cn</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hello, Reyes!</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="StartpushButton">
       <property name="text">
        <string>开始游戏</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="HistorypushButton">
       <property name="text">
        <string>历史记录</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;🐦&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>294</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_G">
    <property name="title">
     <string>游戏（&amp;G）</string>
    </property>
    <addaction name="action_R"/>
    <addaction name="action_S"/>
    <addaction name="addcontentAction"/>
   </widget>
   <widget class="QMenu" name="menu_U">
    <property name="title">
     <string>用户（&amp;U）</string>
    </property>
    <addaction name="action_E"/>
    <addaction name="action_C"/>
    <addaction name="action_X"/>
   </widget>
   <widget class="QMenu" name="menu_O">
    <property name="title">
     <string>其他（&amp;O）</string>
    </property>
    <addaction name="action_A"/>
    <addaction name="action_M"/>
   </widget>
   <addaction name="menu_G"/>
   <addaction name="menu_U"/>
   <addaction name="menu_O"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_R">
   <property name="text">
    <string>查看排行榜（&amp;R）</string>
   </property>
  </action>
  <action name="action_E">
   <property name="text">
    <string>导出用户信息（&amp;E）</string>
   </property>
  </action>
  <action name="action_A">
   <property name="text">
    <string>关于（&amp;A）</string>
   </property>
  </action>
  <action name="action_M">
   <property name="text">
    <string>指南（&amp;M）</string>
   </property>
  </action>
  <action name="action_U">
   <property name="text">
    <string>用户名（&amp;U）</string>
   </property>
  </action>
  <action name="action_P">
   <property name="text">
    <string>密码（&amp;P）</string>
   </property>
  </action>
  <action name="action_S">
   <property name="text">
    <string>设置（&amp;S）</string>
   </property>
  </action>
  <action name="addAcrion">
   <property name="text">
    <string>添加内容</string>
   </property>
  </action>
  <action name="deleteAction">
   <property name="text">
    <string>删除内容</string>
   </property>
  </action>
  <action name="action_X">
   <property name="text">
    <string>删除用户（&amp;X）</string>
   </property>
  </action>
  <action name="addcontentAction">
   <property name="text">
    <string>添加自定义内容（&amp;D）</string>
   </property>
  </action>
  <action name="action_C">
   <property name="text">
    <string>修改信息（&amp;C）</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
